### ISSUE TITLE
what is your performance on squad?

### ISSUE THREAD

**Issue by Jasperty** [RobinRojowiec/qa-answer-selection#1]

**Comment by RobinRojowiec** [comment]
Thank you on your interest in the project.
Depending on the model you're using you can get up to 96% acc with N = 1 for score comparison. For BERT, the problem is you need to have a low learning rate and batch size which is about 32, but this requires at least 12 gb GPU RAM and I wasn't able to access a GPU which has so much RAM.

Following, I'm currently working on a new version of the code and will release a new repository in the near future, which uses the BERT Transformer in a different way and allows training with less RAM.

I keep you posted.
